org.mwc.debrief.core/src/org/mwc/debrief/core/ContextOperations/SelectAllChildren.java

```diff
@@ -188,14 +188,15 @@ public IStatus execute(final IProgressMonitor monitor, final IAdaptable info) th
 
 		protected List<EditableWrapper> itemsFor(final HasEditables parent) {
 			final List<EditableWrapper> res;
```

Comment:
@ayeshare  - have we lost the functionality where this code has been commented out?

Reply:
I have implemented equivalent code for speciaal handling of TacticalDataWrapper now.  But I am not sure how I can test it.